projects/plugins/jetpack/extensions/blocks/contact-form/editor.scss

```diff
@@ -172,6 +171,19 @@ input.components-text-control__input {
 	line-height: 16px;
 }
 
+/*
+	Gutenberg adds some default styling to the TextControl component
+	(https://github.com/WordPress/gutenberg/blob/trunk/packages/base-styles/_mixins.scss#L130)
+	For the TextareaControl, since the base styles are added to the StyledComponent,
+	(https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/textarea-control/styles/textarea-control-styles.ts#L13)
+	The output CSS has a lower specificity and it's being overridden on themes like
+	Twenty Twenty-One and Twenty Twenty-Two.
+	Adding this to normalize the styles between the TextControl and TextareaControl
+ */
+textarea.components-textarea-control__input {
+	border: 1px solid #757575;
+}
```

Comment:
This will override global CSS only when this block is added, right? Is it possible to use a namespace here?

Reply:
Though this border style is to match the default Gutenberg fields style, I think it's safer to add a namespace.
I added it using the `:where` to avoid increasing the selector specificity.